templates/web/kingston/waste/enquiry-problem.html

```diff
@@ -13,10 +13,7 @@
 END;
 
 IF service.report_allowed AND property.domestic_refuse_bin;
-    SET field = {
```

Comment:
If this was meant to make the file a symlink, I don't think it's worked, dunno if there's some git thing going on here.